tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -374,6 +374,12 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.QueryStringReportingEnabled"/>
         public const string QueryStringReportingEnabled = "DD_HTTP_SERVER_TAG_QUERY_STRING";
 
+        /// <summary>
+        /// Configuration key for setting in number of days when to delete log files based on their last writetime date.
+        /// </summary>
+        /// <seealso cref="TracerSettings.MaxLogFilesDays"/>
+        public const string MaxLogFilesDays = "DD_MAX_LOGFILE_DAYS";
```

Comment:
I have two suggestions here:

1. In keeping with the rest of the tracer setting keys, I think we should have the environment variable name start with `DD_TRACE`
2. I thought of a new environment variable name that might describe this feature really well: `DD_TRACE_LOGFILE_RETENTION_DAYS`. WDYT?

Reply:
Missed that, updated it and also found another one we should change at some point.